packages/ember/addon/index.ts

```diff
@@ -67,16 +66,6 @@ export function InitSentryForEmber(_runtimeConfig?: BrowserOptions): void {
   }
 }
 
-/**
- * Grabs active transaction off scope.
- *
- * @deprecated You should not rely on the transaction, but just use `startSpan()` APIs instead.
- */
-export const getActiveTransaction = (): Transaction | undefined => {
```

Comment:
Hmm I guess technically this is public API, users may have imported this? 🤔

Reply:
You're right, we shouldn't break it - changed with 97e08275992ca4295f6686a5d6f89ce802eced3f